### PR TITLE
Reduce travis output - DotReporter

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Adds the DotReporter to codecept so that the Travis output is reduced. It will essentially only display on errors and failures.

## Motivation and Context

Issue reference: #6844 

## How To Test This

See Travis.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by connorshea** [comment]
It doesn't seem like this actually changed the output at all, I wonder if `-vvv` doesn't do what I thought it did?

**Comment by cameronblaikie** [comment]
Assessed :+1:

**Comment by connorshea** [comment]
You'll probably have to remove the `-vvv` verbose flags from each of the commands to make the test suite not output all that extra stuff.

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
       - stage: codecoverage
         if: type IN (pull_request) OR branch in (master, develop, hotfix)
         php: 7.0
-        script: ./vendor/bin/codecept run install --env travis-ci-hub -f -vvv -d; echo "\$sugar_config['state_checker']['test_state_check_mode'] = 1;" >> config_override.php  ; ./vendor/bin/robo -vvv code:coverage; cat codeception.yml; bash <(curl -s https://codecov.io/bash) -f ./_output/coverage.xml
+        script: ./vendor/bin/codecept run install --env travis-ci-hub -f -vvv -d --ext DotReporter; echo "\$sugar_config['state_checker']['test_state_check_mode'] = 1;" >> config_override.php  ; ./vendor/bin/robo -vvv code:coverage; cat codeception.yml; bash <(curl -s https://codecov.io/bash) -f ./_output/coverage.xml
```
Comment (connorshea) [inline .travis.yml]:
```suggestion
        script: ./vendor/bin/codecept run install --env travis-ci-hub -f -d --ext DotReporter; echo "\$sugar_config['state_checker']['test_state_check_mode'] = 1;" >> config_override.php  ; ./vendor/bin/robo -vvv code:coverage; cat codeception.yml; bash <(curl -s https://codecov.io/bash) -f ./_output/coverage.xml
```

File: .travis.yml [hunk 2]
```diff
@@ -67,7 +67,7 @@ before_script:
 script:
   # Run the wizard installer
   - echo "using install wizard"
-  - ./vendor/bin/codecept run install --env travis-ci-hub -f -vvv -d
+  - ./vendor/bin/codecept run install --env travis-ci-hub -f -vvv -d --ext DotReporter
```
Comment (connorshea) [inline .travis.yml]:
```suggestion
  - ./vendor/bin/codecept run install --env travis-ci-hub -f -d --ext DotReporter
```

File: .travis.yml [hunk 3]
```diff
@@ -88,10 +88,10 @@ script:
   - sudo chmod 600 Api/V8/OAuth2/p*.key
   # - sudo chown www-data:www-data Api/V8/OAuth2/p*.key
   # Run API functional tests
-  - ./vendor/bin/codecept run tests/api/V8/ -f -vvv -d
+  - ./vendor/bin/codecept run tests/api/V8/ -f -vvv -d --ext DotReporter
```
Comment (connorshea) [inline .travis.yml]:
```suggestion
  - ./vendor/bin/codecept run tests/api/V8/ -f -d --ext DotReporter
```

File: .travis.yml [hunk 4]
```diff
@@ -88,10 +88,10 @@ script:
...
   # RUN Basic Acceptance test
   - echo "\$sugar_config['imap_test'] = true;" >> config_override.php
-  - ./vendor/bin/codecept run acceptance --env travis-ci-hub -f -vvv -d
+  - ./vendor/bin/codecept run acceptance --env travis-ci-hub -f -vvv -d --ext DotReporter
```
Comment (connorshea) [inline .travis.yml]:
```suggestion
  - ./vendor/bin/codecept run acceptance --env travis-ci-hub -f -d --ext DotReporter
```

**Pull request merged**